dicepy/__init__.py

```diff
@@ -95,7 +51,7 @@ def roll_math(tokens):
 def perform_roll(dice_count: int, sides: int, keep_type: OptionalType[KeepType] = None, keep_amount: int = 1) -> Roll:
     if dice_count > 100 or sides > 100 or keep_amount > dice_count:
         raise Exception('you wot mate?')
-    return Roll(sides, dice_count, keep_type, keep_amount)
+    return Roll(sides, dice_count, None, keep_type, keep_amount)
```

Comment:
Because this reads strangely, I'd rather take advantage of the idea that we can use the named arguments instead of positional ones and do:

```python
return Roll(
    sides=sides,
    dice_count=dice_count,
    keep_type=keep_type,
    keep_amount=keep_amount
)
```

This way, we can avoid the ambiguous `None` in the middle of the arguments and anyone reading the code knows what value is going where and they don't have to care about position.